java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -253,7 +253,7 @@ public static Scalar durationFromLong(DType type, long value) {
         }
         return durationDaysFromInt(intValue);
       } else {
```

Comment:
We at least need a follow on issue to support decimal scalar values.

Reply:
Scalar is mentioned as one of the tasks in umbrella issue https://github.com/rapidsai/cudf/issues/6515. Please let me know if that is okay to track scalar there.

Reply:
That is fine. I mostly want to be sure that we don't forget it.